GNUmakefile

```diff
@@ -72,6 +72,8 @@ ifeq ($(_have-git-version-gen)0,yes$(MAKELEVEL))
         else
           $(info INFO: running autoreconf for new version string: $(_curr-ver))
 GNUmakefile: _version
+	echo oh no
+	sleep 10
```

Comment:
Are these here for debug?

Reply:
yes, I was debugging an issue in the Makefile and forgot to drop these.

Fixed now.  Thanks for noticing it!